modules/ROOT/pages/dataweave-cookbook-extract-data.adoc

```diff
@@ -4,287 +4,200 @@ include::_attributes.adoc[]
+The syntax is `.&amp;myKey`. For examples, see <<key_value>>.
 
-| *Namespace Selector* `keyName.#`
-a| `keyName.#` selector returns the namespace of the selected XML element.
+Descendants (`..`)::
+Acts on arrays and objects to retrieve all matching keys from arrays and
+object below the given key, regardless of their location in the hierarchy.
+The syntax is `..myKey` For more examples, see <<descendants>>.
 
-For an example, see <<namespace>>.
+Index (`[]`)::
+Returns the value at the specified index of an array. An example
+is the `[0]` at the end of `["a","b","c"][0]`, which returns `"a"`. For
+examples, see <<index>>.
 
-| *Selector Modifiers* `?` and `!`
-a| `?` and `!` check for the specified key. `?` returns `true` or `false`.
-`!` returns an error if the key is not present.
+Range `[index1 to index2]`::
+Returns an array with values of the selected indices. An example is
+the `[2 to 3]` at the end of `["a","b","c","d"][2 to 3]`, which returns
+`["c","d"]`. For examples, see <<range>>.
 
-* `{ "a": "b" }.a?` returns `true`. `{ "a": "b" }.c?` returns `false`.
-* `{ "a": "b" }.a!` returns `"b"`. `{ "a": "b" }.c!` produces the error
-`There is no name 'c'`.  `{ "a": "b" }.c` (without `!`) returns `null`.
+XML attribute (`.@myKey`)::
+Returns the value of a selected key for an XML attribute. For an example, see
+<<xml_attribute>>.
 
-For more examples, see <<selector_modifiers>>.
+Namespace Selector (`myKey.#`)::
+Returns the `xmlns` namespace from the element that also contains the selected
+key. For an example, see <<namespace>>.
 
-| *Filter Selectors* `keyName[?(booleanExpression)]`
-a| `keyName[?(booleanExpression)]` returns the selected items if the Boolean
-expression returns `true` and the specified `key` is present. It returns `null`
-if the expression is false or the key is not present.
+Selector Modifiers (`?` and `!`)::
+`?` and `!` check for the specified key. `?` returns `true` or `false`.
+`!` returns an error if the key is not present. For examples, see
+ <<selector_modifiers>>.
 
-For examples, see <<filter_selectors>>.
+Filter Selectors (`myKey[?(booleanExpression)]`)::
+Returns the selected items if the Boolean expression returns `true` and the
+specified `key` is present. It returns `null` if the expression is false or the
+key is not present. For examples, see <<filter_selectors>>.
 
-| *Metadata Selector* `.^someMetadata`
-a| [[selector_metadata]]
+[[selector_metadata]] Metadata Selector `.^someMetadata`::
 Returns the value of specified metadata for a Mule payload, variable, or
-attribute. The selector can return the value of class (`.^class`), encoding
-(`.^encoding`), MIME type (`.^mimeType`), media type  (`.^mediaType`),
-raw  (`.^raw`), and custom (`.^myCustomMetadata`) metadata. See
-<<metadata_selector>>.
+attribute. The selector can return the value of class (`.^class`), content
+length (`.^contentLength`), (`.^encoding`), MIME type (`.^mimeType`),
```

Comment:
You do not have a word before (.^encoding) like you do for the other values.

Reply:
added